app/api/chemotion/attachment_api.rb

```diff
@@ -44,6 +44,90 @@ def upload_chunk_error_message
       error!(message, 404)
     end
 
+    resource :inbox do
+      params do
+        requires :cnt_only, type: Boolean, desc: 'return count number only'
+      end
+      get do
+        inbox_container = Container.find_or_create_by(name: 'inbox', container_type: 'root', containable_id: current_user.id)
+
+        if params[:cnt_only]
+          present inbox_container, with: Entities::InboxEntity, root: :inbox, only: [:inbox_count]
+        else
+          present inbox_container, with: Entities::InboxEntity, root: :inbox
+        end
+      end
+    end
+
+    resource :free_scan do
+      params do
+        requires :count_only, type: Boolean, desc: 'return count number only'
+      end
+      get do
+        free_scan_root_container = Container.find_or_create_by(name: 'free_scan_root', container_type: 'root', containable_id: current_user.id)
+        current_user.container = free_scan_root_container
+
+        free_scans = Entities::FreeScanEntity.represent(free_scan_root_container)
+        if params[:count_only]
+          { inbox: { inbox_count: free_scans.inbox_count } }
+        else
+          free_scans
+        end
+      end
+    end
+
+    resource :attachable do
```

Comment:
Please remove this resource here. It has been moved to the proper Location (aka AttachableAPI)
See https://github.com/ComPlat/chemotion_ELN/blob/development-5/app/api/chemotion/attachable_api.rb